tests/fixtures/textbox.html

```diff
@@ -0,0 +1,5 @@
+<p>
```

Comment:
This is probably going to be bad. I think we need to figure out a solution for this. I suspect we will also want additional test cases that consist of having text within the parent level paragraph as well. For example:

```
<p>
<r><t>Foo</t></r>
<r> ... (textbox) </r>
<r><t>Bar</t></r>
</p>
```

IMO this needs to be de-normalized into three separate paragraphs for HTML purposes.